### PR TITLE
Fix Globalization test on Ubuntu 17.04

### LINKED ISSUE #19284: Test: System.Globalization.Tests.NumberFormatInfoNumberGroupSizes/NumberGroupSizes_Get failed with "Xunit.Sdk.EqualException"
Opened on behalf of @Jiayili1

The test `System.Globalization.Tests.NumberFormatInfoNumberGroupSizes/NumberGroupSizes_Get(format: NumberFormatInfo { CurrencyDecimalDigits = 2, CurrencyDecimalSeparator = \".\", CurrencyGroupSeparator ...` has failed.

Assert.Equal() Failure
    Expected: Int32[] [3, 2]
    Actual:   Int32[] [3]

        Stack Trace:

           at System.Globalization.Tests.NumberFormatInfoNumberGroupSizes.NumberGroupSizes_Get(NumberFormatInfo format, Int32[] expected)
Build : Master - 20170503.01 (Portable Core Tests)
Failing configurations:
- Ubuntu.1704.Amd64-x64
  - Debug
  - Release

Detail: https://mc.dot.net/#/product/netcore/master/source/official~2Fcorefx~2Fmaster~2F/type/test~2Ffunctional~2Fportable~2Fcli~2F/build/20170503.01/workItem/System.Globalization.Tests/analysis/xunit/System.Globalization.Tests.NumberFormatInfoNumberGroupSizes~2FNumberGroupSizes_Get(format:%20NumberFormatInfo%20%7B%20CurrencyDecimalDigits%20=%202,%20CurrencyDecimalSeparator%20=%20%5C%22.%5C%22,%20CurrencyGroupSeparator%20...

### PR DESCRIPTION
**Pull request by tarekgh**
Fixes #19284

### CODE REVIEW

**Comment by tarekgh** [comment]
@danmosemsft could you please have a look?

**Comment by tarekgh** [comment]
@dotnet-bot help

File: src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ public static int[] UrINNumberGroupSizes()
                 ||
                 (PlatformDetection.IsOSX && PlatformDetection.OSXKernelVersion >= new Version(15, 0))
 #endif
-                || PlatformDetection.IsUbuntu1510 || PlatformDetection.IsUbuntu1604 || PlatformDetection.IsUbuntu1610 || PlatformDetection.IsFedora
+                || (PlatformDetection.IsUbuntu && !PlatformDetection.IsUbuntu1404) || PlatformDetection.IsFedora
```
Comment (stephentoub) [inline src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs]: A search for IsUbuntu1604 shows a bunch of other similar conditions in other tests.  Can we fix them all to be more future-proof like this?

Comment (tarekgh) [inline src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs]: I'll do that in another PR.

**Pull request merged**